### PR TITLE
dvm omnibus load option

### PR DESCRIPTION
**Pull request by marcparadise**
Make loading of omnibus components before reconfigure an option. …

- adds new customizable vm.omnibus-auto-load configuration option,
  allowing you to specify the omnibus components you want to have loaded
  before the first reconfigure.
- add simple merge of defaults.yml and config.yml.
- remove vm setting natdnshostresolver1, which causes problems on some
  macbook configurations
- add config.yml to gitignore now that it's more-or-less stable, to make local additions
   less messy

### CODE REVIEW

File: dev/config.yml [hunk 1]
```diff
@@ -1,12 +1,23 @@
-# Override support for default config entries in defaults.yml.
```
Comment (sdelano) [inline dev/config.yml]: Did you mean for this file to be included? It's in the .gitignore

Comment (marcparadise) [inline dev/config.yml]: Yeah, I set up the basic version then separately added it to ignore to lock in place. Alternatively I could use config.yml.example.. .

Comment (marcparadise) [inline dev/config.yml]: I'll go with .example - it's a fairly standard means of doing things.

**Pull request merged**